src/components/node-list/styles/_row-toggle.scss

```diff
@@ -0,0 +1,235 @@
+@import '../../../styles/extends';
+@import '../../../styles/mixins';
+@import '../../../styles/variables';
+@import './variables';
+
+//--- Toggle ---//
+
+.pipeline-row__toggle {
+  position: absolute;
+  right: $row-padding-x;
+
+  &--not-tag {
+    cursor: pointer;
+  }
+
+  &--disabled {
+    display: none;
+  }
+}
+
+@include transparentColour(
+  '--row-selected-transparent',
+  $row-selected-light,
+  $row-selected-dark
+);
+
+.pipeline-row__toggle:before {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: -100px;
```

Comment:
optional: I noticed that we use `em` in some places ( for fonts and margins) and `px` in others, do we normally have any preference on which to use?

Reply:
I would typically use `em` where we want it to be related/relative to font-size, and `px` where it should be absolute.